daemon/mgr/container_types.go

```diff
@@ -49,20 +55,20 @@ func (c *Container) Name() string {
 
 // IsRunning returns container is running or not.
 func (c *Container) IsRunning() bool {
-	return c.meta.Status == types.StatusRunning
+	return c.meta.State.Status == types.StatusRunning
 }
 
 // IsStopped returns container is stopped or not.
 func (c *Container) IsStopped() bool {
-	return c.meta.Status == types.StatusStopped
+	return c.meta.State.Status == types.StatusStopped
 }
 
 // IsCreated returns container is created or not.
 func (c *Container) IsCreated() bool {
-	return c.meta.Status == types.StatusCreated
+	return c.meta.State.Status == types.StatusCreated
 }
 
 // Write writes container's meta data into meta store.
 func (c *Container) Write(store *meta.Store) error {
-	return store.Put(c.meta)
+	return store.Put(c)
 }
```

Comment:
"Container" structure represent a container instance in memory, it don't to be stored. We can add more memory state in "Container", only the meta data need to be stored.

Reply:
OK, I take a look now.

Reply:
Since here meta has a type of `*types.ContainerInfo` in 
```
type Container struct {
	sync.Mutex
	meta       *types.ContainerInfo
	DetachKeys string
}
```

Since types.ContainerInfo is a struct used in API side, so it does not implement "github.com/alibaba/pouch/daemon/meta".Object (missing Key method) . As a result, we need to make this implement.

Reply:
so it...

We define a new type, e.g.:  
type ContainerMeta ContainerInfo. 
func (meta ContainerMeta) Key() string {

}

Reply:
Actually I am doing this in your way.

Reply:
you should use "ContainerMeta" to replace it
https://github.com/alibaba/pouch/blob/master/daemon/daemon.go#L45

Reply:
Updated now, PTAL